Makefile

```diff
@@ -0,0 +1,9 @@
+GO                          := go
```

Comment:
This seems like some aggressive spacing :joy: Unless this is Make-induced-tab-requiredness?

Reply:
lol that's copy pasta spacing.  I'll change that.